src/hotspot/share/jfr/instrumentation/jfrJvmtiAgent.hpp

```diff
@@ -32,10 +32,10 @@ class JfrJvmtiAgent : public JfrCHeapObj {
  private:
   JfrJvmtiAgent();
   ~JfrJvmtiAgent();
-  static bool create();
-  static void destroy();
+  static bool create() NOT_JVMTI_RETURN_(true);
```

Comment:
It initially seemed odd to return `true` here, but looking through the JFR code that interacts with the Agent it seems the right way to view this is that without JVMTI we have a no-op agent.

Reply:
Right.